.github/workflows/CI.yml

```diff
@@ -25,7 +28,12 @@ jobs:
           npx commitlint --version
       - name: Install yarn
         run: |
-          sudo npm install -g yarn
+          npm install -g yarn
+          yarn add --dev jest typescript ts-jest @types/jest
+      - name: Install commitlint
+        run: |
+          npm install conventional-changelog-conventionalcommits
+          npm install commitlint@17.4.2
```

Comment:
@realmarv this is the last thing to fix, it shouldn't specify 17.4.2 because it doesn't work (CI says it's in 17.4.3 anyway). Fix also the commit msg when fixing this.

Reply:
Fixed the version. But I guess the version in the third commit message is correct because I looked at the CI of that commit and the commitlint version is 17.4.2.

Reply:
> I looked at the CI of that commit and the commitlint version is 17.4.2.

what? 🤔 I don't understand that, are you sure you're referring to the last commit?

Reply:
I just double-checked; one job uses 17.4.2 and the other 17.4.3. This is not acceptable, either both use 17.4.2 or both use 17.4.3.

Reply:
Not the last commit. The third commit. I fixed the commit message of the last commit

Reply:
I'm even more confused now, because there are 3 commits in this PR; therefore the last commit is the third commit.

Reply:
Nevermind I guess it's done right?

Reply:
Yes, but being picky, the 3rd commit's title should not just mention "dependencies", because it's upgrading commitlint too.

Reply:
Oh, you fixed that already! nice